build.gradle

```diff
@@ -0,0 +1,137 @@
+import org.springframework.boot.gradle.plugin.SpringBootPlugin
+
+import java.time.Instant
+import java.time.LocalDate
+
+plugins {
+    id 'java'
+    id 'org.springframework.boot' version '2.2.1.RELEASE'
+    id 'io.spring.dependency-management' version '1.0.8.RELEASE'
+    id 'com.github.ben-manes.versions' version '0.27.0'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.50'
```

Comment:
please add 
```
id 'org.owasp.dependencycheck' version '5.2.4'
```
too so that we can run `./gradlew dependencyCheckAnalyze` on the projects to ensure that we're okay with vulnerabilities